Wrappers/WinRT/AnnIndex.idl

```diff
@@ -0,0 +1,37 @@
+namespace SPTAG
+{
+
+  enum LogLevel
+  {
+    Debug = 0,
+    Info,
+    Status,
+    Warning,
+    Error,
+    Assert,
+    Count,
+    Empty
+  };
+
+  [default_interface]
+  runtimeclass SearchResult
+  {
+    Windows.Data.Json.JsonValue Metadata {get; };
```

Comment:
Is it possible just have metadata as blob of bytes here? Algo does not consider the details, library consumer can pick whatever serialization they prefer

Reply:
we could - though that means the onus is on the caller to serialize types, which as you can see isn't exactly easy :)
would it be ok to just add the overload for UInt8[] and let the user pick which data type they want to store?